test/app/model/Pawn.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { Pawn } from '../../../src/app/model/Pawn';
+import { Square } from '../../../src/app/model/Types';
+import 'jest-extended';
+import { boardWithPieces, emptyBoard } from './BoardFixture';
+
+describe('Pawn movement', () => {
+  it("Check possible squares to go, when White Pawn is on D4 and there is no pieces on Pawn's way", () => {
+    const expectedPossibleMoves = [{ column: 'D', row: 5 }];
+    const whitePawn = new Pawn('testId', 'WHITE');
+    const whitePawnPosition: Square = { column: 'D', row: 4 };
+    const whitePawnPossibleMoves = whitePawn.possibleMoves(whitePawnPosition, emptyBoard);
+
+    expect(whitePawnPossibleMoves).toIncludeSameMembers(expectedPossibleMoves);
+  });
+
+  it("Check possible squares to go, when White Pawn is on D7 and there is no pieces on Pawn's way", () => {
+    const expectedPossibleMoves = [{ column: 'D', row: 8 }];
+    const whitePawn = new Pawn('testId', 'WHITE');
+    const whitePawnPosition: Square = { column: 'D', row: 7 };
+    const whitePawnPossibleMoves = whitePawn.possibleMoves(whitePawnPosition, emptyBoard);
+    expect(whitePawnPossibleMoves).toIncludeSameMembers(expectedPossibleMoves);
+  });
+
+  it("Check possible squares to go, when White Pawn is on B4 and there is no pieces on Pawn's way and it's black piece on A5", () => {
+    const testBoard = boardWithPieces({
+      A5: { id: 'testId', side: 'BLACK' },
```

Comment:
Kurde, to id tutaj przeszkadza :/ Moze wywal je od razu? Nie będzie chyba problemu z tym? Bo Paweł chyba mówił, że wyejzdza na weekend, a mam nadzieje, ze kilka figur zostanie zaimplementowanych w tym czasie ;)